store/mockstore/unistore/tikv/mvcc/tikv.go

```diff
@@ -82,16 +82,16 @@ func EncodeWriteCFValue(t WriteType, startTs uint64, shortVal []byte) []byte {
 
 // EncodeLockCFValue encodes the mvcc lock and returns putLock value and putDefault value if exists.
 func EncodeLockCFValue(lock *Lock) ([]byte, []byte) {
-	data := make([]byte, 1)
+	data := make([]byte, 0)
```

Comment:
```suggestion
	var lockTp [1]byte
```
How about allocating this array in the stack?

Reply:
I believe it needs to be a slice anyway.